src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import './setings/general';
+@import './setings/header';
```

Comment:
There is a typo in the import path: 'setings' should be 'settings'. Please correct the folder name in the import statement.